requirements/edx/custom.txt

```diff
@@ -12,9 +13,11 @@
 -e git+https://github.com/edx-solutions/xblock-ooyala.git@v2.0.16#egg=xblock-ooyala==2.0.16
 -e git+https://github.com/edx-solutions/xblock-group-project.git@6a68ea09478e49e796ee4c0a985018ec4257b7d7#egg=xblock-group-project
 -e git+https://github.com/edx-solutions/xblock-adventure.git@7bdeb62b1055377dc04a7b503f7eea8264f5847b#egg=xblock-adventure
--e git+https://github.com/open-craft/xblock-poll.git@1.5.1#egg=xblock-poll==1.5.1
+#-e git+https://github.com/open-craft/xblock-poll.git@v1.6.0#egg=xblock-poll==1.6.0
+-e git+https://github.com/open-craft/xblock-poll.git@cliff/mckin-8752-add-custom-completion#egg=xblock-poll==1.6.0
 -e git+https://github.com/edx/edx-notifications.git@0.6.5#egg=edx-notifications==0.6.5
--e git+https://github.com/open-craft/problem-builder.git@v2.10.5#egg=xblock-problem-builder==2.10.5
+#-e git+https://github.com/open-craft/problem-builder.git@v2.11.0#egg=xblock-problem-builder==2.11.0
+-e git+https://github.com/open-craft/problem-builder.git@cliff/mckin-8751-fix-pb-completion-modes#egg=problem-builder==2.11.0
```

Comment:
Does this need a review as well?

Reply:
@xitij2000 reviewed the problem builder PR.